src/main/charts/bitbucket/Changelog.md

```diff
@@ -112,15 +112,15 @@ index d3342d2..786d343 100644
 * Version 0.9.0 
 * initial commit - changed the max-body-size of request to 250MB for all products and documented in CONFIG.md (#140) 
 * Update values.yaml (#139) 
-* Merge branch 'master' into dckube-267-define-minimum-compute-resources 
+* Merge branch 'main' into dckube-267-define-minimum-compute-resources 
```

Comment:
I am not sure if we should retrospectively change `Changelog.md`, the commits message will stay same.

Reply:
Ive also made this point. Modifying historical changelogs is like modifying the commit history for a repo.